### PR TITLE
NE persisting foods

### PR DESCRIPTION
**Pull request by ski-climb**
@case-eee @icorson3 @mollybrown This persists the newly created foods to localStorage and loads the existing foods from localStorage when the page initially loads.

Basically, yeah! new foods are still on the page when we refresh the page!  Yeah!

### CODE REVIEW

File: lib/foods.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
+var foodList = JSON.parse(localStorage.getItem('foods')) || [];
 
 $('document').ready(function() {
 
+  foodList.forEach(function(element) {
```
Comment (case-eee) [inline lib/foods.js]: love this - i think you could wrap this all in a function though and just call it here 👍

File: lib/foods.js [hunk 2]
```diff
@@ -17,15 +23,11 @@ $('document').ready(function() {
     }
 
     if (foodName.length != 0 && foodCalories.length != 0) {
```
Comment (case-eee) [inline lib/foods.js]: could we possibly break this down into separate functions here?

File: lib/foods.js [hunk 3]
```diff
@@ -17,15 +23,11 @@ $('document').ready(function() {
...
-                    foodEntry +
-                    caloriesEntry +
-                    deleteButton +
-                    "</tr>";
-      $('tbody').prepend(newRow);
+      addFoodToTable(foodName, foodCalories);
+
+      newFood = { name: foodName, calories: foodCalories };
+      foodList.push(newFood);
+      localStorage.setItem('foods', JSON.stringify(foodList));
```
Comment (case-eee) [inline lib/foods.js]: wooooo localStorage!

File: lib/foods.js [hunk 4]
```diff
@@ -67,7 +69,17 @@ $('document').ready(function() {
...
       this.blur();
     }
-
   })
 
+  function addFoodToTable(name, calories) {
+    var foodEntry = "<td contenteditable='true' class='food-name'>" + name + "</td>";
+    var caloriesEntry = "<td contenteditable='true' class='food-calories'>" + calories + "</td>";
+    var deleteButton = "<td class='food-delete'><button>-</button></td>";
+    var newRow = "<tr class='food-row'>" +
```
Comment (case-eee) [inline lib/foods.js]: i like that you've set variables for each of these. it makes it easier to read - i also like that you have a separate function to add the food to the table

File: lib/foods.js [hunk 5]
```diff
@@ -64,7 +64,7 @@ $('document').ready(function() {
     .show();
   })
 
-  $(".food-name, .food-calories").on('keydown', function(event){
+  $("#food-list").on('keydown', ".food-name, .food-calories", function(event){
```
Comment (case-eee) [inline lib/foods.js]: awesome!

**Comment by mollybrown** [comment]
Looks good and thanks for making those small changes.

**Pull request merged**